### PR TITLE
Test files 2022 & 2023 scripts

### PR DESCRIPTION
**Pull request by JBringmannCFPB**

### CODE REVIEW

File: 2023/python/configurations/bank0_config.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+  name:
+    desc: The name of the institution filing data
+    type: String
+    value: Bank0
+  activity_year:
+    desc: The year of activity
+    type: Integer
+    value: 2022
```
Comment (Kibrael) [inline 2023/python/configurations/bank0_config.yaml:8]: should this be 2023?

File: 2023/python/configurations/fake0_config.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+  name:
+    desc: The name of the institution filing data
+    type: String
+    value: "Bank 0"
+  activity_year:
+    desc: The year of activity
+    type: Integer
+    value: 2022
```
Comment (Kibrael) [inline 2023/python/configurations/fake0_config.yaml:8]: change to 2023

File: 2023/python/configurations/newman_bank_config.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+  name:
+    desc: The name of the institution filing data
+    type: String
+    value: Newman Test Bank
+  activity_year:
+    desc: The year of activity
+    type: Integer
+    value: 2022
```
Comment (Kibrael) [inline 2023/python/configurations/newman_bank_config.yaml:8]: change to 2023

File: 2022/python/configurations/bank1_config.yaml [hunk 4]
```diff
@@ -2,10 +2,14 @@
...
     value: Bank1
+  file_length:
+    desc: The number of lines in the clean or test data file
+    type: Integer
+    value: 100
   activity_year:
     desc: The year of activity
     type: Integer
-    value: 2022
+    value: 2023
```
Comment (Kibrael) [inline 2022/python/configurations/bank1_config.yaml:12]: should this be 2022 if it's in the 2022 folder path?

File: 2022/python/configurations/clean_file_config.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@
     activity_year:
       desc: The year of activity
       type: Integer
-      value: 2022
+      value: 2023
```
Comment (Kibrael) [inline 2022/python/configurations/clean_file_config.yaml]: should this be 2022 if it's in the 2022 folder path?

**Comment by zencircle** [comment]
I am confused, I am not seeing `2023` folder in @JBringmannCFPB forked branch but not in master ?

**Pull request closed without merging**